templates/lab-results/templates/main.tpl

```diff
@@ -0,0 +1,144 @@
+{{define "test-result-header"}}
+    <table-cell background-color="#1B7A98" align="center" vertical-align="middle">
+        <paragraph text-align="center">
+            <text-chunk font-size="10" color="#ffffff">{{ .Text }}</text-chunk>
```

Comment:
No need to specify the `font-size` attribute in this case as `10` is the default value.